db/structure.sql

```diff
@@ -0,0 +1,303 @@
+SET statement_timeout = 0;
```

Comment:
do we need this file? is it needed because of the pg_search gem? does this affect the application setup in any way? (like we currently run `rails db:setup` I think and it looks for the schema.rb and seeds) do we need to update the README or something?

Reply:
Yes - I (think I) had to [change the schema dump](https://edgeguides.rubyonrails.org/active_record_migrations.html#types-of-schema-dumps) because `searchable tsvector` doesn't seem to be supported by a Rails schema.

There is some cleanup around this I'd wanna do if we move forward with this approach - but it won't act any differently for users. This loads with the same `rails db:schema:load` because of the change I made here `[config/application.rb](https://github.com/fastruby/points/pull/236/files#diff-c1fd91cb1911a0512578b99f657554526f3e1421decdb9e908712beab57e10f9)`

The upside is we get all of Postgres available to us... downside is it's less friendly to look at than a schema file to see what fields are available. Something like the [Annotate gem](https://github.com/ctran/annotate_models) might help with that.

Reply:
(To be clear I'd wanna remove `db/schema.rb` if I move forward with this since the schema is now held by structure.sql)